### ISSUE TITLE
Merge enum

### ISSUE THREAD

**Issue by Fi1osof** [Urigo/merge-graphql-schemas#179]
Schema:
```
enum MyEnum{
  Value1
}

enum MyEnum{
  Value2
}
```
Result: 
```
enum MyEnum{
  Value1
}
```
enum did not merged.

**Comment by kamilkisiela** [comment]
Should be [fixed](https://github.com/Urigo/merge-graphql-schemas/pull/196/files#diff-ee59d0638e75272d8511d2bc3e6ae758R833) by #196

**Comment by kamilkisiela** [comment]
It's available now under `v1.6.0-beta.2`. Let us know if it works :)
